client-sniffer/build.gradle

```diff
@@ -61,8 +61,11 @@ forbiddenApisTest {
 
 //JarHell is part of es core, which we don't want to pull in
 jarHell.enabled=false
-//NamingConventionCheck is part of test-framework, which we don't want to pull in as it depends on es core
-namingConventions.enabled=false
+
+namingConventions {
+  //we don't have integration tests
+  skipIntegTestInDisguise = true
```

Comment:
I'd prefer settings integTestClass to null or defaulting it to null and setting it our stuff somewhere else in the build.